catalog/transformer/catalog-transformer-streaming-impl/src/main/java/org/codice/ddf/transformer/xml/streaming/impl/Gml3ToWktImpl.java

```diff
@@ -39,17 +39,23 @@
 import org.xml.sax.SAXException;
 
 public class Gml3ToWktImpl implements Gml3ToWkt {
+
   private static final Logger LOGGER = LoggerFactory.getLogger(Gml3ToWkt.class);
 
   private static final String EPSG_4326 = "EPSG:4326";
 
-  private final Parser parser;
+  private final ThreadLocal<Parser> parser;
 
   private static final ThreadLocal<WKTWriter> WKT_WRITER = ThreadLocal.withInitial(WKTWriter::new);
 
   public Gml3ToWktImpl(Configuration gmlConfiguration) {
-    parser = new Parser(gmlConfiguration);
-    parser.setStrict(false);
+    parser =
+        ThreadLocal.withInitial(
```

Comment:
The only downside here is that one of these parser will end up existing in each thread. Depending on how much memory is retained by the parser once it loads all configuration, that might be an issue (I just don't know). Right now it looks like we do have 2 instances of those which means 2 instances of parser per thread.